node/src/components/consensus/era_supervisor.rs

```diff
@@ -966,7 +966,11 @@ impl EraSupervisor {
                 effect_builder
                     .request_appendable_block(block_context.timestamp())
                     .map(move |appendable_block| {
-                        Arc::new(appendable_block.into_block_payload(accusations, random_bit))
+                        Arc::new(appendable_block.into_block_payload(
+                            accusations,
+                            Default::default(/* TODO: fill with the correct value in the next ticket */),
```

Comment:
Do we have this TODO documented in a ticket? Should I add a new one or modify an existing one?

Reply:
That's related to #3912 .

Reply:
Ah, right